corehq/apps/receiverwrapper/models.py

```diff
@@ -337,6 +353,27 @@ def payload_doc(self, repeat_record):
         return None
 
 
+@register_repeater_type
+class JsonFormRepeater(FormRepeater):
```

Comment:
It sets the content type to "application/json". Without that DHIS2 assumes XML, and throws an error when it can't parse it. It also prevents "app_id" from being appended to the URL in API calls.